src/CSharpFunctionalExtensions.FluentAssertions/MaybeExtensions.cs

```diff
@@ -1,8 +1,8 @@
-﻿using FluentAssertions;
```

Comment:
Since you target `net6.0` you can add this to global usings in your `csproj` (or `Directory.Build.props`).

```xml
<ItemGroup>
    <Using Include="CSharpFunctionalExtensions" />
</ItemGroup>
```